packages/server/start-cypress.js

```diff
@@ -1,3 +1,45 @@
+const electronApp = require('./lib/util/electron-app')
+const { telemetry, OTLPTraceExporterCloud } = require('@packages/telemetry')
+const { apiRoutes } = require('./lib/cloud/routes')
+const encryption = require('./lib/cloud/encryption')
+
+// are we in the main node process or the electron process?
+const isRunningElectron = electronApp.isRunning()
+
+const pkg = require('@packages/root')
+
+if (isRunningElectron) {
+  // To pass unencrypted telemetry data to a independent open telemetry endpoint,
+  // disable the encryption header, update the url and add any other required headers.
+  const exporter = new OTLPTraceExporterCloud({
+    url: apiRoutes.telemetry(),
+    // url: 'https://api.honeycomb.io/v1/traces',
+    encryption,
+    // headers: {
+    //   'x-honeycomb-team': 'key',
+    // },
+  })
+
+  telemetry.init({
+    namespace: 'cypress:server',
+    version: pkg.version,
+    exporter,
+  })
+
+  const { debugElapsedTime } = require('./lib/util/performance_benchmark')
+
+  const v8SnapshotStartupTime = debugElapsedTime('v8-snapshot-startup-time')
+  const endTime = v8SnapshotStartupTime + global.cypressServerStartTime
+
+  telemetry.startSpan({ name: 'server', attachType: 'root', active: true, opts: { startTime: global.cypressBinaryStartTime } })
```

Comment:
Would this be better named 'root'?

Reply:
I kind of like `server` as it makes it clear in the telemetry that the server is the orchestrator of everything.

Reply:
i prefer server to root too... server, app, child:process... they're descriptive of where the spans are running.

Reply:
While `server` is descriptive of where the span is running, I don't think it's descriptive of its intention which I feel is more important. For instance, showing `browser` spans under `server` feels odd to me but showing them under `root` feels appropriate.

> ...it makes it clear in the telemetry that the server is the orchestrator of everything.

Why would this be important to someone viewing the telemetry data? If anything, I think we would be explaining to people that even though it shows `server` this is actually all the telemetry data. This feels like an implementation detail if anything.

Reply:
Just to be clear, it would be `cypress:run` or `cypress:open`. If so, then yeah, I like that!

Reply:
yep, you got it!